components/TxForm.tsx

```diff
@@ -127,6 +135,9 @@ export const TxForm = () => {
           Shield
         </Button>
       )}
+      <Button variant="outline" size="lg" mt="1rem" width="100%" onClick={onCopy}>
+        Copy Send Link
```

Comment:
(arbitrary comment) @mds1 @wildmolasses 

I think moving the functionality to where Ed wants makes sense, but instead of text can we use a share logo?

Reply:
is there an icon you have in mind? do you think the functionality would be apparent from the icon, and if not could we pair it with text?

Reply:
Cool I'm good with @wildmolasses's location. For an icon, usually a copy icon looks something like [this](https://fontawesome.com/icons/copy?s=solid&f=classic) (we can use whatever icon library you guys like).

`Copy Send Link <icon>` seems good, I don't love icons with no text because personally I find it confusing UX, unless it tells you what it does with a tooltip on hover

Reply:
I was thinking about this icon, https://fontawesome.com/icons/share-nodes?s=light&f=classic. I am down to keep it with text.